### PR TITLE
Added Ingress Example

### PR DESCRIPTION
**Pull request by ratanboddu**
What this example does : 
   - Creates Deployment
   - Creates Service
   - Creates Ingress

### CODE REVIEW

File: examples/ingress-example.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        namespace="default",
+        body=body
+    )
+
+
+def main():
+    # Fetching and loading local Kubernetes Information
+    config.load_kube_config()
+    # For incluster details
+    # config.load_incluster_config()
```
Comment (yliaog) [inline examples/ingress-example.py]: remove the above commented out line?

**Comment by ratanboddu** [comment]
@yliaog @micw523  I've made the suggested changes.

**Pull request merged**